src/arch/arm/config.cmake

```diff
@@ -13,6 +13,7 @@ endif()
 set(KernelArmPASizeBits40 OFF)
 set(KernelArmPASizeBits44 OFF)
 if(KernelArmCortexA53)
+    set(KernelArmICacheVIPT ON)
```

Comment:
I thought so far that the A35 was the only core that used a VI-PT icache.  I can't find anything about it for the A53.

Reply:
From discussion in mattermost:
https://developer.arm.com/documentation/ddi0500/j/System-Control/AArch64-register-descriptions/Cache-Type-Register?lang=en has the a53 with VI-PT cache.

Reply:
We should probably have a tool that dumps all of those registers for a platform at boot...

Reply:
A53 as well (at least according to the documentation I read; updated the PR with links).

Reply:
Checking that CTR register is as expected at boot would be a good idea, but beyond scope of this PR.